scripts/resolve_bugs.pl

```diff
@@ -42,8 +44,15 @@
 $auto_user->{bless_groups} = [ Bugzilla::Group->get_all ];
 Bugzilla->set_user($auto_user);
 
-my $query = { resolution => '---', product => $product };
-$query->{component} = $component if defined $component;
+if ($buglist) {
+    my $uri = URI->new($buglist);
+    die("Invalid input") if $uri->host ne 'bugzilla.mozilla.org';
```

Comment:
Think about running this with the wrong url, and getting that error. The person running this would probably appreciate getting a more descriptive error message, such as "the buglist url must point to bugzilla.mozilla.org"